### DISCUSSION TITLE
Per AI Save/Load

### DISCUSSION THREAD

**Discussion by kragger** [drazdra/ollama-chats#8] [Ideas]
With a non-persistent interface it is, of course, critical to save and load you state .  I've used it for both base state recycling and the continuation of a longer chat.
I would suggest, though, the ability to save/load an individual AI configuration.  This would be convenient to be able to bring on a base AI in the middle of a session and quickly supply some parameters for an interaction, then remove it once the scene is complete.

**Comment by drazdra** [comment]
i plan adding character cards in future for saving/loading specific characters with their settings.
right now you can only predefine all characters and then turn on stories ui and make several groups. then you can add/remove them to your current group as needed.
